src/tools/shapes/Shape.ts

```diff
@@ -11,24 +14,24 @@ export default class Shape {
 
   getGroup = (): Konva.Group => this.group
 
-  runTemporaryAnimation = (group: Konva.Group, layer: Konva.Layer): void => {
+  runTemporaryAnimation = (group: Konva.Group): void => {
```

Comment:
All tools can use this function (except from the move and erase tool which wouldn't really work that well and is not neccesary. Could be implemented easily though if we want that kind of functionality.)